codalab/server/oauth2_provider.py

```diff
@@ -150,7 +151,7 @@ def validate_client_id(self, client_id):
 
             oauth._validator = MyValidator()
         """
-        expires_in = self.app.config.get('OAUTH2_PROVIDER_TOKEN_EXPIRES_IN')
+        expires_in = self.app.config.get('OAUTH2_PROVIDER_TOKEN_EXPIRES_IN', parse_duration("30d"))
```

Comment:
@teetone It seems like this change doesn't actually have to do with https://github.com/codalab/codalab-worksheets/issues/3888, because that issue doesn't have to do with an expiration. Can you revert this change and see if https://github.com/codalab/codalab-worksheets/issues/3888 is still fixed? In that case, we should then merge this PR in without this file changed.

Reply:
You're right. This change doesn't directly fix #3888, but this change is also related because we use the same auth token to authenticate when the user starts a worker manager. We do want to extend the expiration too.

Reply:
Can you revert this change, then, so we can merge this PR?

And add this change in a separate PR and link it to another issue? The reason is because I think this change is a bit more complicated / requires more discussion before we add it in.